terraformer_policy.tf

```
@@ -11,22 +11,60 @@ data "aws_iam_policy_document" "terraformer_policy_doc" {
 
   # Allow full access to new resources and existing resources that
   # _are not_ tagged as being created by the team that deploys this
-  # root module.
+  # root module, with the exception of IAM.
+  #
+  # We will attach the arn:aws:iam::aws:policy/ReadOnlyAccess policy
+  # to the same role to which policy document will be attached, which
+  # will give is read-only access to all IAM resources.  We require
```

Fix a couple of typos:
```suggestion
  # We will attach the arn:aws:iam::aws:policy/ReadOnlyAccess policy
  # to the same role to which the policy document will be attached, which
  # will give it read-only access to all IAM resources.  We require
```